R/tbl-sql.R

```diff
@@ -13,6 +13,7 @@
 #'   Mainly useful for better performance when creating
 #'   multiple `tbl` objects.
 tbl_sql <- function(subclass, src, from, ..., vars = NULL) {
+  check_dots_used()
```

Comment:
This is not a generic, is this correct?

I'm now seeing failures with duckdb here.